CHANGELOG.md

```diff
@@ -30,38 +34,50 @@ For example:
 @import "node_modules/govuk-frontend/govuk/core/all";
 @import "node_modules/govuk-frontend/govuk/objects/all";
 
-@import "node_modules/govuk-frontend/govuk/components/button";
-@import "node_modules/govuk-frontend/govuk/components/footer";
-@import "node_modules/govuk-frontend/govuk/components/header";
+@import "node_modules/govuk-frontend/govuk/components/button/index";
+@import "node_modules/govuk-frontend/govuk/components/footer/index";
+@import "node_modules/govuk-frontend/govuk/components/header/index";
 ```
 
+Find out more about [importing CSS](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#css).
+
 [Pull request #1804: Allow components to be imported without dependencies](https://github.com/alphagov/govuk-frontend/pull/1804).
 
+#### Back links and breadcrumbs are easier to select
```

Comment:
We're not changing the touch target size of breadcrumbs in this release – it's only back link that's changed.